core/src/main/java/hudson/Functions.java

```diff
@@ -2301,4 +2301,14 @@ public static boolean isContextMenuVisible(Action a) {
             return true;
         }
     }
+
+    /**
+     * Get Locale Display Language depending on the Request's Locale
+     * @param request {@link HttpServletRequest}
+     * @return {@link String} Locale Display Language
+     */
+    public static String getLocale(HttpServletRequest request){
+        return request.getLocale().getDisplayLanguage(request.getLocale());
```

Comment:
I was reading the [comment](https://issues.jenkins-ci.org/browse/JENKINS-62419?focusedCommentId=391363&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-391363), so it basically means that `return new Locale().getLocale().getDisplayLanguage(request.getLocale());` will also work, right?